config/vufind/EDS.ini

```diff
@@ -443,3 +443,20 @@ DetailPageFormat = 'Long'
 ; than you define in ShortAuthorLimit, then VuFind will display the more_authors_abbrev text.
 ; If undefined, the default value is 3.
 ;ShortAuthorLimit = 3
+
+[AdditionalHeaders]
+; Due to the nature of EDS API integrations EBSCO's web application firewall (WAF) does not have sufficient 
+; data to distinguish bots from users. By sending some end-user data to EDS API, EBSCO's WAF can make more 
+; informed decisions regarding bots. 
+; This is important to guarantee accurate COUNTER 5 usage reports. The Counter Code of Practice 5.1 stipulates:
+; "Activity generated by internet robots and crawlers MUST be excluded from all COUNTER usage reports." 
+; See https://cop5.countermetrics.org/en/5.1/07-processing/08-internet-robots-and-crawlers.html
+
+; If you are already protecting your site from bots, e.g. through your own WAF, you might not wish to enable this
+
+; Please note that this will send the end-user's IP Address & User Agent as and x-eis-enduser-ip-address
+; x-eis-enduser-user-agent and you might thus wish to consult your privacy officer.
+
+; send_user_ip = true
+; report_vendor = "VuFind"
+; report_vendor_version = "10.1"
```

Comment:
I would suggest using [VuFind\Config\Version](https://github.com/vufind-org/vufind/blob/dev/module/VuFind/src/VuFind/Config/Version.php) instead of relying on configuration. That way it will be much harder for people to accidentally misreport their version.

Also, a tiny nitpick: I'd recommend adding a line break to the end of this line (we try to end all files with a line break).

Reply:
I had thought of some strange logic to parse the "generator" from config.ini if report_vendor & report_vendor_version are not set. I propose to use the VuFind\Config\Version and remove report_vendor and report_vendor_version and simply hard code VuFind

Reply:
Yes, that seems simplest!